flang/lib/Lower/Bridge.cpp

```diff
@@ -448,14 +448,15 @@ class FirConverter : public Fortran::lower::AbstractConverter {
            cat == Fortran::common::TypeCategory::Complex ||
            cat == Fortran::common::TypeCategory::Logical;
   }
-
   bool isLogicalCategory(Fortran::common::TypeCategory cat) {
     return cat == Fortran::common::TypeCategory::Logical;
   }
-
   bool isCharacterCategory(Fortran::common::TypeCategory cat) {
     return cat == Fortran::common::TypeCategory::Character;
   }
+  bool isDerivedCategory(Fortran::common::TypeCategory cat) {
```

Comment:
Nit: blank lines between function definitions.

Reply:
In `Bridge.cpp` there are small related functions, including overloads and one liners defined without intervening white pace, but also some fairly large non-overloaded functions, such as some of these -

```
  declareFunctions
  declareFunction

  genExprAddr
  genExprValue
  genExprMutableBox

  locationToFilename
  locationToLineno

  getMLIRContext
  mangleName

  defineGlobal
  instantiateGlobal

  insertAggregateStore
  getAggregateStore
  mangleGlobalAgregateStore
  getAggregateType
  defineGlobalAggregateStore
  declreGlobalAggregateStore
  instantiateAggregateStore
  instantiateAlias

  commonBlockHasInit
  getTypeOfCommonWithInitme

  defineCommonBlock
  instantiateCommono

  setCurrentEval
  getEval
```

Is there a preferred style guideline for white space between functions, or is it more of a semi-random choice?